src/hooks/v1/transactor/SafeTransferFromTx.ts

```diff
@@ -12,7 +13,15 @@ export function useSafeTransferFromTx(): TransactorInstance<{
   const { projectId, owner } = useContext(V1ProjectContext)
 
   return ({ newOwnerAddress }, txOpts) => {
-    if (!transactor || !projectId || !contracts?.Projects) {
+    invariant(transactor && contracts?.Projects && projectId && owner)
```

Comment:
**question** I might need more coffee, but is this `invariant` call supposed to be within the `try` block? That way, when `invariant` throws, we're catching it?

Reply:
yesss, good catch.